docs/en/getting-started/example-datasets/amazon-reviews.md

```diff
@@ -92,7 +92,7 @@ ORDER BY (marketplace, review_date, product_category);
 ```sql
 INSERT INTO amazon_reviews
 SELECT
-   * REPLACE(vine = 'Y' AS vine, verified_purchase = 'Y' AS verified_purchase)
+   *
```

Comment:
But I would have Bool in the table - it's easier for queries.

Reply:
Got it - changed to Bool and added a CTE to the INSERT command